tasks/fix-cat2.yml

```diff
@@ -2061,10 +2061,10 @@
       name: "{{ rhel7stig_firewall_service }}"
       state: "{{ rhel7stig_service_started }}"
       enabled: yes
-  # XXX: fix this.  The above task should work just fine in a chroot, but
-  # fails for some reason when the chroot is also a container.
-  ignore_errors: "{{ rhel7stig_system_is_chroot and rhel7stig_system_is_container }}"
-  when: rhel_07_040520
+  when:
+      - rhel_07_040520
+      - not rhel7stig_system_is_chroot
```

Comment:
This will prevent the task from running in a postinstall ks script.  Will firewalld run in a systemd container?

Reply:
It was only 'ignore_errors' when both system_is_chroot and system_is_container were set.  I.e., A non-systemd container.  The other cases worked fine.

Reply:
Ah that was a mistake on my part, I can change it to `and` i just wanted to skip the task rather than ignore errors. Agreed on DISA, NIST already has some guidance but it's quite generic. As far as detecting capabilities at run time, my knowledge falls short, I do not know how you know what caps the container itself has unless your checking from the docker host itself to be able to do `getcap` or `netcap` or similar. Ideas?

Reply:
`capsh --print` might be a good start... not sure, and unfortunately I don't have time now to find out.

Reply:
ah, TIL, that would work. I just tested.